src/models/m_rsc_update.erl

```diff
@@ -53,11 +53,10 @@
 
 
 %% @doc Insert a new resource. Crashes when insertion is not allowed.
--spec insert(list(), #context{}) -> {ok, integer()}.
+-spec insert(m_rsc:props(), #context{}) -> {ok, m_rsc:resource_id()}.
```

Comment:
The spec in line 56 uses a record, please define a type for the record and use that instead.